src/components/Dashboard.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import Display from "./Display";
+
+class Dashboard extends Component {
+  constructor() {
+    super();
+    this.state = {
+      strikes: 0,
+      balls: 0
+    };
+  }
+
+  strike = () => {
+    if (this.state.strikes === 2) {
+      this.setState({ strikes: 0, balls: 0 });
+    } else {
+      this.setState({ strikes: this.state.strikes + 1 });
+    }
+  };
+
+  ball = () => {
+    if (this.state.balls === 3) {
+      this.setState({ balls: 0, strikes: 0 });
+    } else {
+      this.setState({ balls: this.state.balls + 1 });
+    }
+  };
+
+  hit = () => {
+    this.setState({ strikes: 0, balls: 0 });
+  };
+
+  foul = () => {
+    if (this.state.strikes <= 1) {
+      this.setState({ strikes: this.state.strikes + 1 });
+    } else return;
```

Comment:
This `else return` seems unnecessary as this function will return `undefined` regardless if the if condition is false.